src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);

```

Comment:
<img width="995" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/81781479/51d6054e-85ce-45a2-90ba-0983f335a071">

make Random and ColorSupplier class level variables

avoid magic numbers ✨ use constants